.github/ISSUE_TEMPLATE/creating-a-new-release.md

```diff
@@ -56,6 +56,9 @@ git push --delete upstream <tag_name>
 git clean -x -d -n
 # before running the next command check, that it is ok to remove the files
 git clean -x -d -f
+# Please remove python3-setuptools_scm, or it will all git files into tarbal.
+# Please unset PYTHONPATH variable, or it will include unexpected files into
+# tarball.
 python3 setup.py sdist
```

Comment:
This should probably do this:
```suggestion
PYTHONPATH= python3 setup.py sdist
```

Reply:
Fixed by `env --unset=PYTHONPATH python3 setup.py sdist`